apps/api-extractor/src/generators/packageTypings/PackageTypingsGenerator.ts

```diff
@@ -269,6 +324,64 @@ export class PackageTypingsGenerator {
     return this._entriesBySymbol.get(followAliasesResult.followedSymbol);
   }
 
+  // NOTE: THIS IS A TEMPORARY HACK.
+  // In the near future we will overhaul the AEDoc parser to separate syntactic/semantic analysis,
+  // at which point this will be wired up to the same ApiDocumentation layer used for the API Review files
+  private _getReleaseTagForSymbol(symbol: ts.Symbol): ReleaseTag {
+    const fullyFollowedSymbol: ts.Symbol = TypeScriptHelpers.followAliases(symbol, this._typeChecker);
+
+    let releaseTag: ReleaseTag = ReleaseTag.None;
+
+    const releaseTagRegExp: RegExp = /(?:\s|\*)@(internal|alpha|beta|public)/g;
```

Comment:
>) [](start = 75, length = 1)

We probably want whitespace or a line ending after the tag.

Reply:
Good catch - fixed

---
In reply to: [172382182](https://github.com/Microsoft/web-build-tools/pull/556#discussion_r172382182) [](ancestors = 172382182)